app/src/main/java/com/github/se/eventradar/model/event/EventCategory.kt

```diff
@@ -1,18 +1,36 @@
 package com.github.se.eventradar.model.event
 
+import android.content.Context
+import android.util.Log
+import com.github.se.eventradar.R
+
 enum class EventCategory(val displayName: String) {
   MUSIC("Music"),
   SPORTS("Sports"),
   CONFERENCE("Conference"),
   EXHIBITION("Exhibition"),
   COMMUNITY("Community"),
   SOCIAL("Social"),
+  PARTY("Party");
+
+  fun convertToResString(context: Context): String {
```

Comment:
Still don't think this is necessary. It adds more complexity to the implementation when one can just use the enum.

Reply:
I cannot reply under your other comments so I'll answer here

1. I can agree that it is of no use in V1 of our app. But as a french speaker, I would like to have the app in french lol, and we wouldn't put every translation in the enum. This is why I implemented this using the resource string. But if you're strongly against having that function in V1, I can remove it.

2. Colors are exactly the same as the one already used in the rest of the app. I use the default colorScheme with "onSurface" "onSurface Variant" etc... I just didn't hard coded the color scheme in my individual UI components. We can discuss the relevance of this. Because at that point in the app, colorScheme colors could be hard coded in the components since they always take the same primary material and we have currently no use to switch them to secondary color scheme.